org.vergecurrency.verge-qt.json

```diff
@@ -0,0 +1,69 @@
+{
+  "id": "org.vergecurrency.verge-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "6.0.2",
```

Comment:
This is the version of runtime specified in `runtime`. You probably want to set it to `19.08`.

Reply:
resolved